<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Digvijayt-VD3" failures="0" tests="1" name="DataSupplier.ExcelData_Supplier" time="0.000" errors="1" timestamp="2022-11-16T14:19:09 IST" skipped="0">
  <testcase classname="DataSupplier.ExcelData_Supplier" name="testdata" time="0.000">
    <error message="org.joor.ReflectException: java.lang.reflect.InvocationTargetException" type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException: org.joor.ReflectException: java.lang.reflect.InvocationTargetException
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:57)
at org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:157)
at org.testng.internal.Parameters.handleParameters(Parameters.java:791)
at org.testng.internal.Parameters.handleParameters(Parameters.java:728)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:917)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.joor.ReflectException: java.lang.reflect.InvocationTargetException
at org.joor.Reflect.on(Reflect.java:914)
at org.joor.Reflect.call(Reflect.java:575)
at io.github.sskorol.utils.ReflectionUtils.invokeDataSupplier(ReflectionUtils.java:72)
at io.github.sskorol.model.DataSupplierMetaData.obtainReturnValue(DataSupplierMetaData.java:56)
at io.github.sskorol.model.DataSupplierMetaData.transform(DataSupplierMetaData.java:40)
at io.github.sskorol.model.DataSupplierMetaData.<init>(DataSupplierMetaData.java:36)
at io.github.sskorol.core.DataProviderTransformer.getMetaData(DataProviderTransformer.java:64)
at io.github.sskorol.core.DataProviderTransformer.supplySeqData(DataProviderTransformer.java:23)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:53)
... 25 more
Caused by: java.lang.reflect.InvocationTargetException
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.joor.Reflect.on(Reflect.java:910)
... 39 more
Caused by: org.joor.ReflectException: java.lang.reflect.InvocationTargetException
at org.joor.Reflect.on(Reflect.java:914)
at org.joor.Reflect.call(Reflect.java:575)
at org.joor.Reflect.call(Reflect.java:528)
at io.github.sskorol.data.TestDataReader$DataBuilder.read(TestDataReader.java:54)
at DataSupplier.ExcelData_Supplier.getdata(ExcelData_Supplier.java:17)
... 44 more
Caused by: java.lang.reflect.InvocationTargetException
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.joor.Reflect.on(Reflect.java:910)
... 48 more
Caused by: java.lang.IllegalArgumentException: Unable to read XLSX data to class DataSupplier.ExcelPojo.
at io.github.sskorol.data.XlsxReader.read(XlsxReader.java:68)
... 53 more
Caused by: java.lang.IllegalArgumentException: Column username not found
at io.github.sskorol.data.XlsxCellMapper.lambda$new$0(XlsxCellMapper.java:40)
at java.base/java.util.Optional.orElseThrow(Optional.java:403)
at io.github.sskorol.data.XlsxCellMapper.<init>(XlsxCellMapper.java:40)
at io.github.sskorol.data.XlsxReader.lambda$read$3(XlsxReader.java:60)
at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:992)
at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575)
at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260)
at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616)
at one.util.streamex.AbstractStreamEx.toArray(AbstractStreamEx.java:390)
at one.util.streamex.AbstractStreamEx.toMutableList(AbstractStreamEx.java:1328)
at one.util.streamex.AbstractStreamEx.toList(AbstractStreamEx.java:1309)
at io.github.sskorol.data.XlsxReader.read(XlsxReader.java:61)
... 53 more
]]>
    </error>
  </testcase> <!-- testdata -->
  <system-out/>
</testsuite> <!-- DataSupplier.ExcelData_Supplier -->
